### PR TITLE
ArchivistsToolkitClient: use pymysql instead of MySQLdb

### PR DESCRIPTION
**Pull request by mistydemeo**
A brief test suggests this works fine, and works in Python 3.

### CODE REVIEW

**Comment by sevein** [comment]
Nice!

File: agentarchives/archivists_toolkit/client.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
 import logging
 
+import pymysql
+pymysql.install_as_MySQLdb()
+
 import MySQLdb
```
Comment (Hwesta) [inline agentarchives/archivists_toolkit/client.py]: Does this stay?

Comment (mistydemeo) [inline agentarchives/archivists_toolkit/client.py]: Yes; `pymysql.install_as_MySQLdb()` doesn't create a module in the local namespace called `MySQLdb`, but instead allows `import MySQLdb` to load `pymysql`.

Comment (mistydemeo) [inline agentarchives/archivists_toolkit/client.py]: However it looks like it doesn't do anything more than register a fake `MySQLdb` module to point to `pymysql`, so changed this to just `import pymysql as MySQLdb`.

**Comment by Hwesta** [comment]
:+1:

**Pull request merged**